dev/com.ibm.tx.jta/resources/com/ibm/ws/Transaction/resources/TransactionMsgs.nlsprops

```diff
@@ -883,6 +883,13 @@ WTRN0155_CHECKPOINT_NEW_TX_STACK=WTRN0155W: An application began or required a t
 WTRN0155_CHECKPOINT_NEW_TX_STACK.explanation=The stack trace was captured when the transaction was created. The stack trace helps users determine which applications begin transactions during application startup and then cause the checkpoint request to fail.
 WTRN0155_CHECKPOINT_NEW_TX_STACK.useraction=Examine the stack trace to determine which application began a new transaction during the server checkpoint request.
 
+WTRN0156_CANNOT_RESUME=WTRN0156E: Transaction cannot be resumed on the current thread because it is active on a different thread ({0}).
```

Comment:
In this message and the one after, start the message with "The transaction" rather than "Transaction"